deploy/charts/harvester/dependency_charts/harvester-network-controller/values.yaml

```diff
@@ -26,3 +26,43 @@ tolerations:
     effect: NoSchedule
 
 affinity: {}
+
+
+# Default value for harvester-network-controller-manager deployment
+replicas: 3
+
+strategy:
+  rollingUpdate:
+    maxSurge: 1
```

Comment:
why do we intend to make the max surge of manager pod to be 1?

Reply:
It was added by @futuretea. It's optional and I will delete it.